src/legends/swatches.js

```diff
@@ -93,6 +93,14 @@ function legendItems(scale, {
         --swatchHeight: ${+swatchHeight}px;
       `);
 
+  const palette = label
+      ? swatches.call(div => div.append("p")
+          .text(label)
+          .style("font-weight", "bold")
+          .style("width", "100%"))
```

Comment:
The parent element here has flex because the code expects that it contains the swatches. We need to change the styles to match the structure—we shouldn’t be setting the width style.